assets/js/modules/adsense/components/dashboard/AdSenseConnectCTAWidget.js

```diff
@@ -98,15 +98,14 @@ function AdSenseConnectCTAWidget( { Widget, WidgetNull } ) {
 		);
 	}
 
-	if ( adSenseModuleConnected || hasDismissedWidget ) {
-		return <WidgetNull />;
-	}
-
-	return (
+	// Checking if === false explicitly, as they can be undefined
+	if ( adSenseModuleConnected === false && hasDismissedWidget === false ) {
 		<Widget noPadding>
 			<AdSenseConnectCTA onDismissModule={ showTooltip } />
-		</Widget>
-	);
+		</Widget>;
+	}
```

Comment:
@sashadoes, we have to return the JSX. 
```js
if ( adSenseModuleConnected === false && hasDismissedWidget === false ) {
		return (
			<Widget noPadding>
				<AdSenseConnectCTA onDismissModule={ showTooltip } />
			</Widget>
		);
	}
```

Otherwise, it doesn't render the `Widget`. Due to this, the tests are failing as well.

Reply:
I have fixed it in this [commit](https://github.com/google/site-kit-wp/pull/6050/commits/51b95a31e48d08ca1dc4f573a9d31c83754c89f0).